test/connection_test.py

```diff
@@ -0,0 +1,138 @@
+# Databricks notebook source
```

Comment:
I'm not sure what GetConnection or does for me that build_connection does not.

Reply:
really the read_write_test covers if connection_test works